README.md

```diff
@@ -1,2 +1,10 @@
 # infoblox-cloud-init
 opensource cloud-init project 
+
```

Comment:
I don't see any info about the 2 files we have 

How to use it?
How to integrate with cloud-init packages etc?

Do we have any unit test?

How to package this?